pages/admin/lessons/new/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+import { gql, useLazyQuery } from '@apollo/client'
+import { AdminLessonCard } from '../../../../components/admin/lessons/AdminLessonCard'
+import { withGetApp, GetAppProps } from '../../../../graphql'
+import { AdminLayout } from '../../../../components/admin/AdminLayout'
+import Link from 'next/link'
+import styles from '../../../../scss/adminLessonPage.module.scss'
+
+const EXERCISES = gql`
+  query {
+    exercises {
+      flaggedAt
+      module {
+        lesson {
+          title
+        }
+      }
+    }
+  }
+`
+type Exercise = {
+  flaggedAt?: string
+  module: {
+    lesson: {
+      title: string
+    }
+  }
+}
+
+type Exercises = Exercise[]
+
+const Lessons: React.FC<GetAppProps> = ({ data }) => {
+  const { lessons } = data
+  const [lessonCards, setCards] = useState<[]>([])
+  const [getExercises, { data: exercisesData }] = useLazyQuery<{
+    exercises: Exercises
+  }>(EXERCISES, {
+    onCompleted: async () => {
+      const exerciseMapping = await exercisesData?.exercises.reduce(
+        (prev: any, cur) => {
+          if (cur.flaggedAt) {
+            if (!prev[cur.module?.lesson?.title]) {
+              prev[cur.module?.lesson?.title] = 1
+              return prev
+            }
+            prev[cur.module?.lesson?.title]++
+          }
+
+          return prev
+        },
+        {}
+      )
+
+      setCards(
+        lessons?.reduce((prev: any, cur: any, i) => {
+          prev.push(
+            <div key={i} className={styles.lessonCard}>
+              <AdminLessonCard
+                lesson={cur}
+                pendingFlaggedQuestions={
+                  exerciseMapping[cur.title] ? exerciseMapping[cur.title] : 'No'
```

Comment:
`exerciseMapping[cur.title]` is repeated twice, make it into a variable

Reply:
Actually, instead of a making variable, it would be better to simplify this to `exerciseMapping[cur.title] || 'No'`, since your use of ternary operator is doing what the logical operator `||` already does